test/truefi2/lines-of-credit/TrueCreditAgency.test.ts

```diff
@@ -430,6 +445,11 @@ describe('TrueCreditAgency', () => {
       expect(await creditAgency.nextInterestRepayTime(tusdPool.address, borrower.address)).to.eq(timestamp.add(MONTH))
     })
 
+    it('borrows and check if mutex is locked', async () => {
```

Comment:
This name doesn't sound right to me. Maybe `borrowing locks mutex`, but I'm open to other suggestions as well.